test/puppetlabs/jdbc_util/pool_test.clj

```diff
@@ -140,8 +141,29 @@
   (let [test-pool (-> core-test/test-db
                       pool/spec->hikari-options
                       pool/options->hikari-config
-                      (pool/connection-pool-with-delayed-init identity 5000))]
+                      (pool/connection-pool-with-delayed-init {:migration-db core-test/test-db} identity 5000))]
     (.getConnection test-pool)
     (is (= {:state :ready}
            (pool/status test-pool)))
     (.close test-pool)))
+
+(deftest migration-db-spec-test
```

Comment:
While running this test I get the exception `org.postgresql.util.PSQLException: FATAL: role "migrator" does not exist`, but the tests all pass. Is this normal?